drivers/hud_apr/app/models/hud_apr/generators/shared/cell_details_concern.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudApr::CellDetailsConcern
+  extend ActiveSupport::Concern
+
+  included do
+    private def common_headings
+      [
+        :first_name,
+        :last_name,
+      ].freeze
+    end
+    helper_method :common_headings
+  end
+end
```

Comment:
Is this used?  (Again, I think this should go into the models)

Reply:
✅  Yeah, that was just cruft. Still a WIP.